triangulo.py

```diff
@@ -0,0 +1,32 @@
+ladoa = int(input('Digite o valor do Lado A:\n'))
+while ladoa < 1:
+    print('Digite um valor maior que 0.')
+    ladoa = int(input('Digite o valor do Lado A:\n'))
+
+ladob = int(input('Digite o valor do Lado B:\n'))
+while ladob < 1:
+    print('Digite um valor maior que 0.')
+    ladob = int(input('Digite o valor do Lado B:\n'))
+
+ladoc = int(input('Digite o Valor do Lado C:\n'))
+while ladoc < 1:
+    print('Digite um valor maior que 0.')
+    ladoc = int(input('Digite o Valor do Lado C:\n'))
+
+v1 = (ladoa + ladob)
+v2 = (ladoa + ladoc)
+v3 = (ladob + ladoc)
+
+if ladoa < v3:
+    resultado = True
```

Comment:
Pra quesito de leitura do codigo, que tal mudar o nome da variabel para **eUmTriangulo** ou algo similar? 
Acredito que ficaria mais facil de entender do que somente _resultado_.

Reply:
Eu tenho que acostumar com a ideia que outra pessoa vai ler o código já agora no início, verdade.